flowmachine/flowmachine/features/location/labelled_spatial_aggregate.py

```diff
@@ -0,0 +1,116 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from typing import List
+
+from flowmachine.core import Query
+from flowmachine.core.mixins import GeoDataMixin
+from flowmachine.features.location.spatial_aggregate import SpatialAggregate
+
+
+class LabelledSpatialAggregate(GeoDataMixin, Query):
+    """
+    Class representing a disaggregation of a SpatialAggregate by some label or set of labels
+
+    Parameters
+    ----------
+    locations: Query
+        Any query with a subscriber and location columns
+    subscriber_labels: Query
+        Any query with a subscriber column
+    label_columns: List[str]
+        A list of columns in subscriber_labels to aggregate on
+
+    Example:
```

Comment:
```suggestion
    Examples
    --------
```
(as per [numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html#examples))